### DISCUSSION TITLE
Neuberechnung der Strecke

### DISCUSSION THREAD

**Discussion by ghost** [osmandapp/OsmAnd#20576] [Ideas]
Wenn man die Strecke neu berechnen möchte, muss man aktuell auf einen Wegpunkt klicken und anschließend auf Routentyp ändern.
Dann erscheint nach Scrollen der Hinweis "Route neu berechnen".
Besser wäre es, diese Funktion in die Optionen zu verlagern.

Bei längeren Strecken baut OsmAnd manchmal Fehler in den Track ein. Nach Neuberechnung ist der Fehler dann wieder behoben. Aber Zugang zur Funktion ist umständlich.

**Comment by DmitryAlexei** [comment]
You can attach your track to the roads:
- after scrolling the navigation menu up (navigation > scroll that menu up)
- in the follow track menu (navigation > settings > follow track menu > attach to the roads)

Please see documentation https://osmand.net/docs/user/navigation/setup/gpx-navigation#attach-to-the-roads

**Reply by ghost** [reply]
Ich meine diese Funktion
<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/e30ba85a-768a-4269-acb0-4bc2b3dd8772"  width="300" height="auto"/> </td>
  </tr>
</table>

**Comment by ghost** [comment]
Wie ich schon sagte, es kommt vor das OsmAnd im Editiermodus die Linien Kreuz und quer zieht. Klickt man auf die o.g. Funktion, passt es wieder.

**Reply by DmitryAlexei** [reply]
To reproduce that issue, we will need the coordinates of the starting and end points and profiles you use (can be attached in .zip file to github via menu > settings > profile > export profile)
